### PR TITLE
Template for Keynotes, 2024 edition

### PR DESCRIPTION
**Pull request by bvdmitri**
This PR creates a template for keynotes on the website.

What is missing so far

- [ ] Photo for Andrew
- [ ] Photo for Sam
- [ ] Biography for Michael
- [ ] Biography for Rachel
- [ ] Biography for Andrew
- [ ] Biography for Sam
- [ ] Extra keynotes, which weren't confirmed yet (how many?)
- [x] Titles? e.g. Dr or Prof.

Overall, the template looks like

Screenshot 2024-05-17 at 22 06 22

### CODE REVIEW

File: 2024/index.md [hunk 1]
```diff
@@ -21,16 +21,55 @@ top_title_heading = "../assets/2021/img/world_768.png 768w, ../assets/2021/img/w
...
   <div style="text-align: center">
-    <h2 class="btn" style="width: 60%; border-color: #9d253d;"><a style="color: #9d253d; text-decoration: none;" href="https://www.tickettailor.com/events/numfocus1/1114995#">Buy your ticket here!</a></h2>
+    <h2 class="btn" style="width: 60%; border-color: #854c9e;"><a style="color: #854c9e; text-decoration: none;" href="https://www.tickettailor.com/events/numfocus1/1114995#">Buy your ticket here!</a></h2>
   </div>
   ~~~
 \end{box}
 
+\begin{box}{title="Keynote Speakers", color="red"}
+
+  \avatar{name="Prof.Dr. Bert de Vries", affil="Eindhoven Technical University", link="https://biaslab.github.io/", img="/assets/2024/img/keynotes/pbert.png"}
```
Comment (ranjanan) [inline 2024/index.md:36]: Does he go by Prof. Dr. ?

File: 2024/index.md [hunk 2]
```diff
@@ -21,16 +21,55 @@ top_title_heading = "../assets/2021/img/world_768.png 768w, ../assets/2021/img/w
...
+
+  @@content-box-talk
+    Large language model-based chatbots such as chatGPT are very impressive, but you cannot ask them to go out and learn how to ride a bike. Learning how to ride a bike is about an agent that learns a skill through efficient, real-time interactions with a dynamic environment. In this presentation, I will discuss the underlying technology that enables brains to learn new skills and acquire knowledge solely through unsupervised environmental interactions. How much do we understand about what brains compute? And is this knowledge transferable to AI engineering systems? I will discuss Karl Friston’s Free Energy Principle, which is the theory on what, why and how brains compute. I will also discuss the efforts of our research lab (http://biaslab.org) at TU Eindhoven to develop a Julia ecosystem of packages to support transfer of these ideas to working AI engineering tools.
+  @@
+
+  ---
+
+  \avatar{name="Michael F. Herbst", affil="Mathematics and Institute of Materials Science, École polytechnique fédérale de Lausanne, Switzerland", link="https://michael-herbst.com/", img="/assets/2024/img/keynotes/michael.jpg"}
```
Comment (ranjanan) [inline 2024/index.md:44]: Prof.

File: 2024/index.md [hunk 3]
```diff
@@ -21,16 +21,55 @@ top_title_heading = "../assets/2021/img/world_768.png 768w, ../assets/2021/img/w
...
+
+  \avatar{name="Rachel Kurchin", affil="Materials Science and Engineering department at Carnegie Mellon University in Pittsburgh, Pennsylvania, USA", link="https://engineering.cmu.edu/directory/bios/kurchin-rachel.html", img="/assets/2024/img/keynotes/rachel.png"}
```
Comment (ranjanan) [inline 2024/index.md:46]: Prof.

File: 2024/index.md [hunk 4]
```diff
@@ -21,16 +21,55 @@ top_title_heading = "../assets/2021/img/world_768.png 768w, ../assets/2021/img/w
...
+
+  @@content-box-talk
+    Current solutions to key 21st-century challenges (such as climate change, food insecurity, healthcare, and communications) are fundamentally limited by the functional properties of known materials. Designing new materials increasingly relies on computational modeling with state-of-the-art workflows frequently interweaving first-principles with empirical modeling as well as data-driven approaches. To make advances, we often must establish novel connections across fields such as physics and chemistry.
+  @@
+
+  ---
+
+  \avatar{name="Andrew Fitzgibbon", affil="TBF", link="TBF", img="/assets/2024/img/world_768.png"}
```
Comment (ranjanan) [inline 2024/index.md:54]: Dr.

File: 2024/index.md [hunk 5]
```diff
@@ -21,16 +21,55 @@ top_title_heading = "../assets/2021/img/world_768.png 768w, ../assets/2021/img/w
...
+
+  @@content-box-talk
+    AI is fast becoming a significant consumer of the world’s computational power, so it is crucial to use that power wisely and efficiently. Our approaches to doing so must span all levels of the research stack: from fundamental theoretical understanding of the loss surfaces and regularization properties of machine learning models, to efficient layout at the transistor level of floating-point multipliers and RAM.
+  @@
+
+  --- 
+
+  \avatar{name="Sam Aaron", affil="TBF", link="TBF", img="/assets/2024/img/world_768.png"}
```
Comment (ranjanan) [inline 2024/index.md:62]: Dr.

File: 2024/keynotes.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Natural Artificial Intelligence
+
+Large language model-based chatbots such as chatGPT are very impressive, but you cannot ask them to go out and learn how to ride a bike. Learning how to ride a bike is about an agent that learns a skill through efficient, real-time interactions with a dynamic environment. In this presentation, I will discuss the underlying technology that enables brains to learn new skills and acquire knowledge solely through unsupervised environmental interactions. How much do we understand about what brains compute? And is this knowledge transferable to AI engineering systems? I will discuss Karl Friston’s Free Energy Principle, which is the theory on what, why and how brains compute. I will also discuss the efforts of our research lab (http://biaslab.org) at TU Eindhoven to develop a Julia ecosystem of packages to support transfer of these ideas to working AI engineering tools.
+
+---
+
+\avatar{name="Michael F. Herbst", affil="Mathematics and Institute of Materials Science, École polytechnique fédérale de Lausanne, Switzerland", link="https://michael-herbst.com/", img="/assets/2024/img/keynotes/michael.jpg"}
```
Comment (ranjanan) [inline 2024/keynotes.md:13]: Prof.

File: 2024/keynotes.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+\avatar{name="Rachel Kurchin", affil="Materials Science and Engineering department at Carnegie Mellon University in Pittsburgh, Pennsylvania, USA", link="https://engineering.cmu.edu/directory/bios/kurchin-rachel.html", img="/assets/2024/img/keynotes/rachel.png"}
```
Comment (ranjanan) [inline 2024/keynotes.md:15]: Prof.

File: 2024/keynotes.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+--- 
+
+\avatar{name="Sam Aaron", affil="TBF", link="TBF", img="/assets/2024/img/world_768.png"}
```
Comment (ranjanan) [inline 2024/keynotes.md:31]: Dr.

File: 2024/keynotes.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+---
+
+\avatar{name="Andrew Fitzgibbon", affil="TBF", link="TBF", img="/assets/2024/img/world_768.png"}
```
Comment (ranjanan) [inline 2024/keynotes.md:23]: Dr. 

and affiliation is Graphcore. Ref: https://www.fitzgibbon.ie/

File: 2024/keynotes.md (hunk 8 above)
Comment (ranjanan) [reply 2024/keynotes.md:31]: Affiliation is Sonic Pi. Website is: https://x.com/i/flow/login?redirect_after_login=%2Fsamaaron.

Comment (ranjanan) [reply 2024/keynotes.md:31]: Or: http://sam.aaron.name/

**Review by ranjanan — CHANGES_REQUESTED** [review]

File: 2024/index.md (hunk 1 above)
Comment (bvdmitri) [reply 2024/index.md:36]: Yes, its actually even longer `Prof.Dr.Ir.` but I will double check with him what does he prefer

**Review by ranjanan — APPROVED** [review]

**Pull request merged**